utils.py

```diff
@@ -27,6 +28,10 @@ def getCVE_Dict(year: str) -> {}:
     :return: CVE dictionary
     """
```

Comment:
InstalledSoftware: method and function name should be camel_case too (`connect_to_mainKey`)